### DISCUSSION TITLE
Master Unlocker 1.36.0

### DISCUSSION THREAD

**Discussion by jhowkNx** [jhowkNx/Master-Unlocker#46] [Announcements]

**Comment by brain-vs-zombie** [comment]
About selection and setup of mission vehicles for selling mission in CEO Warehouse, I start the mission in the warehouse according to the requirements and selected the corresponding vehicle, but the actual mission requires the use of other vehicle  (For example, I have chosen the tugboat as the mission vehicle for selling mission, but when I started the selling, the actual vehicle may not be the tugboat, maybe the mission vehicle was a plane or three tracks) so what are the reason for this situation (maybe I start the misson in private? or other reasons?) and how should I solve this problem.

**Reply by DyingRhythmYT** [reply]
just a quick question, are there any precautions i should use and if so dose anyone have a link or video on what not to do with the script/menu

**Reply by jhowkNx** [reply]
> About selection and setup of mission vehicles for selling mission in CEO Warehouse, I start the mission in the warehouse according to the requirements and selected the corresponding vehicle, but the actual mission requires the use of other vehicle (For example, I have chosen the tugboat as the mission vehicle for selling mission, but when I started the selling, the actual vehicle may not be the tugboat, maybe the mission vehicle was a plane or three tracks) so what are the reason for this situation (maybe I start the misson in private? or other reasons?) and how should I solve this problem.

I believe this bug happens more because of the amount of items you are going to sell, it's a known bug. It will be fixed as soon as there is a definitive fix. 



> just a quick question, are there any precautions i should use and if so dose anyone have a link or video on what not to do with the script/menu

Just get enough, not what you want. Any function that changes money, increases money is risky, now dealing with unlocks, R* doesn't care about that.

**Comment by liquidwater01** [comment]
Hello, it seems the fill special cargo warehouse with single purchase option only works with large warehouses. It does not work with medium (have not tested small). I was hoping to fill medium warehouses with the special item so that the sale price would be less than 10M. Is this expected?

**Comment by liquidwater01** [comment]
Special cargo instant finish does not work when the mission involves going to pick up satchel charges then swimming to blow open containers under water. Upon exiting the office and after the phone call, activate Instant Finish Special Cargo (Buy) and the mission fails saying the cargo was not delivered.

**Comment by ham1tcagdas** [comment]
Possible ? if this T-Shirt can be unlocked in master unl.
![x](https://user-images.githubusercontent.com/101466850/207185086-a3451619-09a1-45ad-9b55-eb6d52dcc9d2.png)

**Reply by jhowkNx** [reply]
hmm this feature has been in Master Unlocker for a long time.
`Unlocks > Clothings > Special Tops`
![Captura de tela 2022-12-13 005532](https://user-images.githubusercontent.com/79918850/207222900-9339a34c-5f5b-4800-92c1-add8e3b94eca.png)
